src/BSeries.jl

```diff
@@ -296,6 +299,43 @@ for op in (:+, :-)
     end
 end
 
+# investigate properties of B-series
+"""
+    order_of_accuracy(series; kwargs...)
+
+Determine the order of accuracy of the B-series `series`. By default, the
+comparison with the coefficients of the exact solution is performed using
+`isequal`. If keyword arguments such as absolute/relative tolerances `atol`/`rtol`
+are given or floating point numbers are used, the comparison is perfomed using
+`isapprox` and the keyword arguments `kwargs...` are forwarded.
+
+See also [`order`](@ref), [`ExactSolution`](@ref).
+"""
+function order_of_accuracy(series::TruncatedBSeries; kwargs...)
+    if isempty(kwargs) && !(valtype(series) <: AbstractFloat)
+        compare = isequal
+    else
+        compare = (a, b) -> isapprox(a, b; kwargs...)
+    end
+
+    exact = ExactSolution{valtype(series)}()
+
+    # TODO: TruncatedBSeries
+    # This assumes that the `coef` are always stored with increasing `order` of the
```

Comment:
Why not go with the following approach?  
- Loop over order (starting from 1).
- Generate all trees of that order.
- Check coefficients of each tree.

I guess it would be slower, but generating all trees is still quite fast, right?

Reply:
Yeah, right. That might just be better. Let me take a look.

Reply:
Done in 9544a9a